hw/ip/spi_host/dv/spi_host_sim_faster_prescaler.hjson

```diff
@@ -0,0 +1,32 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  variant: faster_prescaler
+
+  // Import the base spi_device sim_cfg file
+  import_cfgs: ["{proj_root}/hw/ip/spi_host/dv/base_sim_cfg.hjson"]
+
+  // Enable the appropriate build mode for all tests
+  build_opts: ["+define+FAST_PRESCALER=1"]
```

Comment:
Rather than splitting the config, I think you can do this by adding an extra build mode. I think you can:
- Add an extra "build mode" called something like fast_prescaler
- This build mode should add the option in question
- Then use that build mode in the test by selecting it with `build_mode`.

Untested, but I think you can do something like add this near the top o the file
```
  build_modes: [
    {
      name: fast_prescaler
      build_opts: ["+define+FAST_PRESCALER=1"]
    }
  ]
```

Then the test can be something like
```
    {
      name: spi_host_upper_range_clkdiv
      uvm_test_seq: spi_host_upper_range_clkdiv_vseq
      reseed: 10 // Reduced reseed since this test may take ages
      build_mode: "fast_prescaler"
    }
```

Reply:
Thanks - I'm going to give this ago. 
Can you elaborate on what this does? 
Does this mean DVSIM has two different snapshots? One with the `define set and the other one without?